tsbridge/config_test.go

```diff
@@ -40,25 +44,32 @@ func TestNewConfigSimple(t *testing.T) {
 		t.Errorf("cfg.metrics expected to have 4 elements; got %v", cfg.metrics)
 	}
 
-	// 'testapp' is the default app id used by aetest.
+	// 'testapp' is the default app id used by the emulator
 	if cfg.StackdriverDestinations[0].ProjectID != "testapp" {
 		t.Errorf("expected destination project to be equal to app id; got %v", cfg.StackdriverDestinations[0].ProjectID)
 	}
 
 	// project_id parameter is required when app id cannot be detected.
-	for _, appid := range []string{"", "None"} {
-		appIDFunc = fakeAppIDFunc(appid)
-		_, err := NewConfig(testCtx, &ConfigOptions{Filename: "testdata/valid.yaml"})
-		if !strings.Contains(err.Error(), "please provide project_id for") {
-			t.Errorf("passing project_id should be required")
-		}
+	fakeProjectID := ""
```

Comment:
I don't understand why you're creating a local variable for an empty string, when it's never changed.

Reply:
Fixed.